js/addToTask.js

```diff
@@ -0,0 +1,74 @@
+import {
+  createTaskButton,
+  content,
+  list,
+  inputTask,
+  input,
+  addTask,
+  trashInput,
+  today,
+  COMPLETE,
+  EDIT,
+  DELET,
+  CLICK_EVENT,
+  HIDE,
+} from "/js/constants.js"; //named import
+
+export let LIST = [];
+export let id = 0;
+let tarikh;
+function addToDo(toDo, id, done, edit, trash) {
```

Comment:
```suggestion
function addTodo(todo, id, done, edit, trash) {
```
Besides I see no use of `done, edit `inside the function then why are we taking these as parameters?

Reply:
yeah, it was redundant.